sale_stock_line_customer_ref/models/stock_move.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    customer_ref = fields.Char(
```

Comment:
Wondering: shall we add computed non-stored field on the move line?
IMO it helps in many cases.

Reply:
I was wondering too, put a related on `stock.move.line` to know what we are referring to.